website/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@graphprotocol/docs",
+  "version": "1.0.0",
+  "packageManager": "pnpm@7.28.0",
```

Comment:
I guess we don't need it here since we already specified it in root package.json